src/pymovements/datasets/dataset.py

```diff
@@ -346,8 +366,15 @@ def load_event_files(self, events_dirname: str | None = None) -> list[EventDataF
             filepath = Path(fileinfo['filepath'])
             filepath = self.raw_rootpath / filepath
 
-            filepath = self._raw_to_event_filepath(filepath, events_dirname=events_dirname)
-            event_df = pl.read_ipc(filepath)
+            filepath = self._raw_to_event_filepath(
+                filepath, events_dirname=events_dirname,
+                extension=extension,
+            )
+
+            if extension == 'feather':
+                event_df = pl.read_ipc(filepath)
+            elif extension == 'csv':
+                event_df = pl.read_csv(filepath)
```

Comment:
we need also need an additional `else` case, otherwise the extension 'unknown' will lead to a confusing error in the next line (as `event_df` won't be available)

raise a `ValueError` in the `else` clause. use a text like this:
```
f'unsupported file format "{extension}". Supported formats are: {valid_extensions}'
```
with
```
valid_extensions = ['csv', 'feather']
```

Reply:
done